module/Application/config/module.config.php

```diff
@@ -23,6 +23,18 @@
                 ],
                 'priority' => 100
             ],
+            'teapot' => [
+                'type' => 'Literal',
+                'options' => [
+                    'route' => '/teapot',
```

Comment:
The correct route for this would be `/coffee` I believe

Reply:
You're entirely right